src/repository.ts

```diff
@@ -331,7 +330,9 @@ export class LernaRepo extends Repository {
   public verifySignature(packageNames: string[]): void {
     const packages = this.packages.filter((pkg) => packageNames.includes(pkg.name));
     for (const pkg of packages) {
-      const cmd = `sfdx-trust plugins:trust:verify --npm ${pkg.name}@${pkg.getNextVersion()}`;
+      const cmd = `sfdx-trust plugins:trust:verify --npm ${
+        pkg.name
+      }@${pkg.getNextVersion()}  ${this.registry.getRegistryParameter()}`;
```

Comment:
Now, there will always be a `--registry` flag, and it'll default to regular npm registry because we're relying on the user to create the npm config file?

Registry() accepts a `registryUrl` to its constructor...wondering if we should make the default registry the default for Registry().

I'm imagining someone stubbing their toe sometime with this when they get `--registry undefined` especially since we don't have tests for that.

Reply:
@mshanemc The Registry class is constructed [here](https://github.com/salesforcecli/plugin-release-management/blob/phale/sfdx-trust-registry/src/registry.ts#L20) in Repository. Registry [ctor](https://github.com/salesforcecli/plugin-release-management/blob/phale/sfdx-trust-registry/src/registry.ts#L20) calls init and loadNpmConfigs. The init function sets registry from env var, if available and loadNpmConfigs set registry from npm config or defaults it to public registry URL.